module_utils/kafka_manager.py

```diff
@@ -1328,9 +1330,28 @@ def alter_quotas(self, quotas):
                             key: str(value)
                             for key, value in configs.items()
                         })
-                        self.zk_client.create(znode, bytes(str(json.dumps({
-                            'version': 1,
-                            'config': configs
-                        })).encode('ascii')), makepath=True)
+                        self.zk_client.create(base_znode + znode,
+                                              bytes(str(json.dumps({
```

Comment:
Not really link to this PR, but it makes me thing we may need to change our `json.dumps(var).encode('ascii')` to `json.dumps(var, ensure_ascii=False).encode('utf-8')` so that an user with some emojis our whatever chars in the `user` name, for instance, will be fine

Reply:
Good idea. Will make an issue to deal with all occur ;)